api/v1alpha/framework/imagerunner.schema.json

```diff
@@ -4,6 +4,9 @@
   "description": "Configuration file for running container images using saucectl",
   "type": "object",
   "allOf": [
+    {
+      "$ref": "../subschema/sauce.schema.json"
```

Comment:
But the spec would be lying then, since only a subset is supported. Isn't the general idea of the validation to be helpful? So this would show you a bunch of values that you can't really apply to the imagerunner.

We likely need a separate definition for `sauce` in imagerunner. Open to any other ideas 🤷

Reply:
I also hesitated about it. Is there any possibility that we'll add more fields into imagerunner? But I can add the separate `sauce` firstly

Reply:
> Is there any possibility that we'll add more fields into imagerunner

Possibility? Oh yeah, for sure. There may be also capabilities that are added later down the road that are specific to the imagerunner and nowhere else. Can neither confirm nor deny whether and when it's going to happen though.

Reply:
Let's just keep this separated `sauce` definition and improve it later